src/modules/random-number.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const randomFourDigits = () => Math.floor(1000 + Math.random() * 9000);
```

Comment:
how about repetitive numbers?
e.g. `0.555 * 9000 + 1000 = 5995` - would be nice so that numbers do not repat
but not crucial